apps/condo/pages/user/index.tsx

```diff
@@ -146,14 +176,32 @@ export const UserInfoPageContent: React.FC<IUserInfoPageContentProps> = ({ organ
                                                                     options={possibleLocalesOptions}
                                                                     value={locale}
                                                                     placeholder={ChooseInterfaceLanguageTitle}
-                                                                    onChange={localeChangeHandler(setLocale)}
+                                                                    onChange={handleLocaleChange(setLocale)}
                                                                 />
                                                             )
                                                         }}
                                                     </LocaleContext.Consumer>
                                                 </Col>
                                             </Row>
                                         </Col>
+                                        {locale === 'ru' && (
```

Comment:
Why it doesn't support different locales?

Reply:
hints are visible only for the `ru` locale. So no need to show hint settings for other locales - that would be strange